graph/task.go

```diff
@@ -24,6 +24,16 @@ const (
 
 	// The default step retry delay is 5 seconds.
 	defaultStepRetryDelayInSeconds = 5
+
+	// currentTaskVersion is the most recent Task version.
+	currentTaskVersion = "v1.0.0"
```

Comment:
Does it require to add `v` prefix for semantic version?

Reply:
Nah, from FAQ of SemVer:

> Is "v1.2.3" a semantic version?
> No, "v1.2.3" is not a semantic version. However, prefixing a semantic version with a "v" is a common way (in English) to indicate it is a version number. Abbreviating "version" as "v" is often seen with version control. Example: git tag v1.2.3 -m "Release version 1.2.3", in which case "v1.2.3" is a tag name and the semantic version is "1.2.3".
> 

I think it makes sense to have the v prefix here though, since it's consistent with other projects and we can re-use it for the git tag (+automatic linking during compile time via the existing `version` tag), and the vx.x.x is the pattern for vgo/modules.

Reply:
https://github.com/golang/go/wiki/Modules#version-selection